### ISSUE TITLE
AndroidWiFiADB connects to device, but it isn't displayed as deployment target

### ISSUE THREAD

**Issue by RomiValladares** [pedrovgs/AndroidWiFiADB#60]
Hi.
I've successfully used the plugin before, but after updating Android Studio it doesn't work anymore.

Android Wifi ADB version: 2.4
Android Studio version: 2.3.3
JRE: 1.8.0

Steps
1. Check that both computer and device are connected to the same Wi-Fi.
2. Connect device via USB; Android Studio recognizes it.
3. Connect with Android WiFi ADB, disconnect USB.
4. Try to run the app; the device isn't shown on the deployment targets dialog.
5. Disconnect from Android WiFi ADB; an error is shown: 'Unable to disconnect device, make sure that your computer and your device are connected to the same network'.

[**I've uploaded a video showing the steps.**](https://vimeo.com/223328935)

**Comment by SykesMod** [comment]
happened to me too
as a workaround until this gets fixed, try this workaround:

phone:
enable ADB over network in developer options, take note of the ip (default port 5555)

android studio (windows):
then go to terminal tab of android studio and navigate to your adb folder
(by default, its in this path):
C:\Users\**your_username***\AppData\Local\Android\sdk\platform-tools

once there, simply issue this command
adb.exe connect <ip-from-your-phone>

for a different android studio platform, just go to terminal and issue the adb connect command

**Comment by RomiValladares** [comment]
Thanks! It didn't work though. Maybe it wasn't a problem with the AndroidWiFiADB, and there's something else going on, even though I have the Mac and the device connected to the same WiFi.
<img width="435" alt="screen shot 2017-07-03 at 10 35 46 pm" src="https://user-images.githubusercontent.com/8409964/27811776-174b6086-6040-11e7-9a32-907b91098cac.png">

**Comment by SykesMod** [comment]
try disabling AndroidWifiADB plugin and issuing commands again

**Comment by pedrovgs** [comment]
Hi @SykesMod thanks for reporting! I can't reproduce this bug using the same Android Studio version in my computer 😥  It seems you can easily reproduce it! It could be great if you debug the bug and send a PR I tried installing different Android Studio version and using different devices but the plugin seems to be working as expected 😢

**Comment by RohitK1992** [comment]
Getting this same issue in Android Studio 3.0.1 stable version.

**Comment by kailash09dabhi** [comment]
Yes On my mac it shows connected but dont show up "Select Deployment Target" window!

**Comment by Tokimu** [comment]
Same on my mac, android studio 3.0.1

**Comment by piyan13** [comment]
Same on my linuxmint , android studio 3.0.1

**Comment by guitorioadar** [comment]
I think the problem is with the port.
I found a way from this [link](https://androvaid.com/debug-apps-in-real-device-over-wifi/). If it might help.
Open cmd and write  `$ adb devices` , that will show you the list of connected devices. your device must be connected via a usb cable.
`$ adb tcpip 5555` , write it in the cli and it will restart port: 5555.
`$ adb connect 192.168.0.102:5555`
